lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -500,8 +519,9 @@ class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
           if (failed(minimallyCollapseDimHelper(
                   op, rewriter, inputDim, nextUnchangedInput, outputDim,
                   nextUnchangedOutput, inputShapeVec, outputShape,
-                  outputAssociations.back())))
+                  inputAssociations.back(), outputAssociations.back()))) {
             return failure();
+          }
           hasDynamic = false;
           outputDim = outputAssociations.back().back() + 1;
           inputDim++;
```

Comment:
I think you need to be properly updating the inputDim here the same way the outputDim is incremented. Ditto for the other call to minimallyCollapseDimHelper.